test/policies/proxy/proxy.test.js

```diff
@@ -0,0 +1,174 @@
+const path = require('path');
+const fs = require('fs');
+const request = require('supertest');
+const assert = require('chai').assert;
+const https = require('https');
+const express = require('express');
+const expressApp = express();
+
+const config = require('../../../lib/config');
+const gateway = require('../../../lib/gateway');
+
+const originalGatewayConfig = config.gatewayConfig;
+const port = 5998;
+
+const serverKeyFile = path.join(__dirname, '../../fixtures/certs/server', 'server.key');
+const serverCertFile = path.join(__dirname, '../../fixtures/certs/server', 'server.crt');
+const incorrectClientKeyFile = path.join(__dirname, '../../fixtures', 'agent1-cert.key');
+const clientKeyFile = path.join(__dirname, '../../fixtures/certs/client', 'client.key');
+const clientCertFile = path.join(__dirname, '../../fixtures/certs/client', 'client.crt');
+const chainFile = path.join(__dirname, '../../fixtures/certs/chain', 'chain.pem');
+
+describe('proxy policy', () => {
+  let app, service;
+
+  before('start service', () => {
+    expressApp.all('*', function (req, res) {
+      res.status(200).json({ port: port }).end();
+    });
+
+    service = https.createServer({
```

Comment:
`server`, `httpServer` are probably better names

Reply:
Shall change to server.